package.json

```diff
@@ -82,6 +82,7 @@
     "body-parser": "1.19.0",
     "bull": "3.20.1",
     "bull-board": "1.3.1",
+    "clean-whitespace": "0.1.2",
```

Comment:
What's the point of this? Ah, from the merge?

Reply:
Yup, it's cleaned out now :)